### PR TITLE
fix: add active as well as focus for Safari

### PR DESCRIPTION
**Pull request by apalski**
## Enable NakedButton outline in Safari

Safari doesn't implement `focus` so I've added the same css behaviour to `active`.

💪

## Looks like this

https://user-images.githubusercontent.com/21995514/160057132-38e80f50-c175-49fb-800b-15daff64d93e.mov

https://user-images.githubusercontent.com/21995514/160057140-98c63434-55db-413d-b3bb-6ef6517f9f8f.mov

🤨

---

**PR check-list**

🚨 **IMPORTANT:** Please review the [guidelines for contributing](./CONTRIBUTING.md) to this repository first! 👀

- [ ] You've requested a review from at least ONE person. Preferably a roo-ui maintainer ([Philip](https://github.com/philipwindeyer), [Eric](https://github.com/eneo5541), [Mike](https://github.com/memcc), [Marcel](https://github.com/Anthropic), [Sam](https://github.com/SamuSan) [Long](https://github.com/KieraDOG)) or a Roo UI working-group human.
  - Note: If your PR has not been reviewed within two days of requesting a review, feel free to seek reviews from other [team members of Qantas Hotels](https://github.com/orgs/hooroo/people).
- [ ] Your final merge commit message is in the format `<type>(<scope>): <description>` as per [the contributing guide](https://github.com/hooroo/roo-ui/blob/master/.github/CONTRIBUTING.md#commit-naming)
  - Example: `fix(dependencies): minor version bump to resolve security vulnerability in set-value`

### CODE REVIEW

File: src/components/NakedButton/NakedButton.js [hunk 1]
```diff
@@ -33,6 +33,10 @@ const NakedButton = styled('button')`
     outline: ${themeGet('borders.2')} ${themeGet('colors.brand.secondary')};
   }
 
+  &:active {
```
Comment (eneo5541) [inline src/components/NakedButton/NakedButton.js:36]: `active` and `focus` can be combined into a single css style:
```
&:focus, &:active {
    outline: ${themeGet('borders.2')} ${themeGet('colors.brand.secondary')};
  }
```

**Review by eneo5541 — APPROVED** [review]
LGTM! Just one minor styling fix

File: src/components/NakedButton/NakedButton.js (hunk 1 above)
Comment (apalski) [reply src/components/NakedButton/NakedButton.js:36]: Ah nice! Thanks @eneo5541 😄

**Review by philipwindeyer — APPROVED** [review]
🚀  (after updating snapshots 😛 )

**Pull request merged**